samples/matter/light_bulb/CMakeLists.txt

```
@@ -6,7 +6,11 @@
 
 cmake_minimum_required(VERSION 3.20.0)
 
-set(APPLICATION_CONFIG_DIR "configuration/\${BOARD}")
+set(MATTER_MODULE_ROOT $ENV{ZEPHYR_BASE}/../modules/lib/matter)
```

```suggestion
get_filename_component(MATTER_MODULE_ROOT $ENV{ZEPHYR_BASE}/../modules/lib/matter REALPATH)
```
to use shorter paths.